server/src/main/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponse.java

```diff
@@ -12,47 +14,58 @@
 import io.micronaut.data.annotation.AutoPopulated;
 import io.micronaut.data.annotation.TypeDef;
 import io.micronaut.data.model.DataType;
+import io.swagger.v3.oas.annotations.media.Schema;
 
 @Entity
 @Table(name="pulse_response")
 public class PulseResponse {
-    
-    public PulseResponse() {}
 
-    public PulseResponse(LocalDate submissionDate, LocalDate updatedDate, UUID teamMemberId, String internalFeelings, String externalFeelings) {
-        this.submissionDate = submissionDate;
-        this.updatedDate = updatedDate;
-        this.teamMemberId = teamMemberId;
-        this.internalFeelings = internalFeelings;
-        this.externalFeelings = externalFeelings;
-    }
-    
     @Id
     @Column(name="id")
     @AutoPopulated
     @TypeDef(type=DataType.STRING)
+    @Schema(description = "the id of the pulse_response", required = true)
     private UUID id;
 
     @Column(name="submissionDate")
-    @NotNull
+    @Schema(description = "id of the submissionDate", required = true)
```

Comment:
If these fields are required for the api marked by the Schema annotation required=true, then these should all have NotNull annotated on them.

Reply:
Added NotNull to schema fields. thanks.